### ISSUE TITLE
Change warning for WMC track with less than 20 streamlines

### ISSUE THREAD

**Issue by DanNBullock** [brainlife/brainlife#2]
Currently, it seems that any WMC with less than 20 streamlines triggers a warning on the interface.  While this is sensible for something like wma-seg, which corresponds to established tracks that ought to be fairly substantial, there are other wmc-related apps that this is not advisable for.  For example, in the category segmentation (https://doi.org/10.25663/brainlife.app.249) it is quite likely that some categories will have a small number of streamlines, and this is not a problem.  I would recommend having this warning as an option for app managers, rather than the default for all wmc apps.

**Comment by soichih** [comment]
Currently, each App that generates wmc has to produce `product.json`, tracts/surfaces for vis, and validate and generate some warnings like the message "Some tracts have less than 20 streamlines". 

I will be working on a wmc datatype validator that can do all of that, rather than having each Apps do it. Once this is in place, then we can drop the functions like "generate_productjson.m" - along with the me "less than 20 streamlines" message.

> https://github.com/search?q=generate_productjson.m&type=Code

For app-wmaSeg, it sounds like we still want to display this message. So for app-wmaSeg, we will need to keep the stopped down version of generate_productjson.m that generates this message in product.json which will be merged into the product.json from the validator.
